### ISSUE TITLE
RSS/Atom feed seems badly broken

### ISSUE THREAD

**Issue by siliconforks** [smashingmagazine/redesign#264]
**- Do you want to request a *feature* or report a *bug*?**

Bug.

**- What is the current behavior?**

The RSS/Atom feed contains lots of weird stuff, like dates of 01 Jan 0001, dates from 2018, etc.

**- If the current behavior is a bug, please provide the steps to reproduce.**

Visit https://www.smashingmagazine.com/index.xml and check the `pubDate` values

**- What is the expected behavior?**

I'm not really sure what the intended behavior is here - whether these items are even supposed to be in the feed at all.

**- Please mention your operating system version and the version of your browser**

N/A

**Comment by vitalyfriedman** [comment]
Dear @siliconforks — could you double check again, please? The issue should be resolved now.

**Comment by maiki** [comment]
@vitalyfriedman the feed at https://www.smashingmagazine.com/feed/ works, but you ought to turn off the feed at https://www.smashingmagazine.com/index.xml, which shows weird content from all over the place (space *and* time).

**Comment by lembitk** [comment]
I discovered the new, broken [`index.xml` feed](https://www.smashingmagazine.com/index.xml) because the [old, “canonical” `feed/` feed](https://www.smashingmagazine.com/feed/) was redirected to it for some time around the recent relaunch (~9 days, considering when this issue was opened..?). 

The redirect is gone now which IMHO solves the issue for most folks (i.e. most of pre-relaunch RSS subscribers) who just experienced the interim redirect in their RSS readers. 

However, there might be people who subscribed directly to the new `index.xml` feed recently, and some of them might be/get confused about messages from the future and other weird stuff in their readers :) So the `index.xml` feed (and other Hugo's feeds) might still need some further care. (I am quite sure the new `index.xml` feed was given as _the_ RSS link in site's source around the relaunch – I checked, as was thinking about reporting the same issue myself.) 

P.S. It seems #261 is somewhat related to the issue at hand…
